vexim/config/functions.php

```diff
@@ -127,6 +127,28 @@ function crypt_password($clear, $salt = '')
                     }
                     $cryptedpass = crypt($clear, $salt);
                 break;
+		case 'sha512':
```

Comment:
This would leave the user with more configuration options. But we still have fixed salt-lengths in our code, for SHA512 you can even add more parameters `$6$rounds=5000$salt$` that needs all to be compatible. We would have to test first if all these schemes are compatible with courier and dovecot (even cyrus?).